Japanese localisation for Yakyak

jefu7 opened the pull request:
Hello. I'm proposing these localization strings to create a Japanese localized version of Yakyak. I am new to this kind of work, so will be grateful for technical guidance.  (I'll take care of editing the Japanese per suggestions, which are welcome!)

averissimo commented on src/locales/jp.json:
```diff
@@ -0,0 +1,172 @@
+                "newer": "新しいバージョンが利用可能です(%s から a %s)。",
+                "license": "ライセンス",
+                "authors": "主な作者",
+                "contributors": "貢献者"
```
There's a string missing:

"startup": "Open on startup"

See line 163 of https://github.com/yakyak/yakyak/blob/a16f24c09fffa4f8d6596f4258b47d4d1c382f92/src/locales/en.json.template#L163

jefu7 commented:
Thanks.  And sorry about that.

I will find a way to translate it... and try to figure out how to add it. It was a small miracle that I was able to get it to you on the first place!

Let's see what I can do...

Thanks.

Jeff


On Sep 12, 2017, at 11:38, André Veríssimo <notifications@github.com> wrote:

@averissimo requested changes on this pull request.

Please add the missing string :) If by any chance there's no translation keep it in English

Thanks

In src/locales/jp.json:

> +                "extra_large": "極大"
+            },
+            "zoom": {
+                "in": "ズームイン",
+                "out": "ズームアウト",
+                "reset": "標準"
+            }
+        },
+        "help": {
+            "title": "ヘルプ",
+            "about": {
+                "title": "ヤックヤックについて",
+                "newer": "新しいバージョンが利用可能です(%s から a %s)。",
+                "license": "ライセンス",
+                "authors": "主な作者",
+                "contributors": "貢献者"
There's a string missing:

"startup": "Open on startup"

See line 163 of https://github.com/yakyak/yakyak/blob/a16f24c09fffa4f8d6596f4258b47d4d1c382f92/src/locales/en.json.template#L163

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.

averissimo commented:
Not a miracle!! everything is right.

You probably started working on the translation before we added that string.

Once the pull request is updated with that string (i.e. you change it, commit and push) then we'll merge it

jefu7 commented:
Thanks, André.  I appreciate your patience, very much.

I’ve added it that new line, and then I hit “Commit”. I can’t find anything that will let me “Push”, though.

Hope it works!

Jeff

On Sep 12, 2017, at 2:02 PM, André Veríssimo <notifications@github.com> wrote:

Not a miracle!! everything is right.

You probably started working on the translation before we added that string.

Once the pull request is updated with that string (i.e. you change it, commit and push) then we'll merge it

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/yakyak/yakyak/pull/756#issuecomment-328983609>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AeEh2Nbo5bdrWlwPnvhfIGr4uBT4XLDxks5shvGDgaJpZM4PQkyZ>.

averissimo commented:
merged to master :)

jefu7 commented:
Wow. I am delighted! Thank you for guiding me so kindly, André.

Now I can encourage Japanese friends and colleagues to use Yakyak.

Thanks again!

Jeff

averissimo commented:
I had to made some small corrections: 

- locale name for japan is `ja` and not `jp`
- there were some strange " on it that I had to replace

It is now updated and working on master branch.

Added the japanese translation to the pre-release v1.4.4 :) go ahead and test it out and tell your friends :)

jefu7 commented:
Dear André:  Thanks very much!  And I'm sorry about the local name mistake.  The " problems might have been that they somehow got entered as full-width characters even though I tried hard to avoid that.

I'm glad it all works now.  It will be great to have this in Japanese!

It's been fun working on this. Please let me know how I can help further if any feedback comes back regarding my translations.

Take care.

Jeff
